microsite/data/plugins/tech-insights.yaml

```diff
@@ -1,15 +1,15 @@
 ---
 title: Tech Insights
 author: roadie.io
-authorUrl: https://github.com/RoadieHQ
+authorUrl: https://github.com/backstage-community
```

Comment:
I think you accidentally got the wrong url here!

Reply:
Hi @hdost, could you check the comment above so we can merge the pull request 🙂 ?